main.py

```diff
@@ -182,16 +187,7 @@ def make(self):
             if crashed:                                   # Game crash or Close check
                 pygame.quit()
                 sys.exit()
-
-        # Just a window exception check condition
-        event1 = pygame.event.get()
-        if event1.type == pygame.QUIT:
-            crashed = True
-
-        if crashed:
-            pygame.quit()
-            sys.exit()
-
+        
```

Comment:
Removed the part where events were dequeued for the second time.

Reply:
Yet you have not mentioned that by doing this you are fixing https://github.com/sugarlabs/flappy-birds-activity/issues/21

Reply:
I mentioned it in the pull request title.

Reply:
Yes. I know.  But we don't keep the pull request title in git.  We keep the commit messages.  That's why commit messages are to describe and communicate your work.